apps/providers/lib/providers/timestamp.ex

```diff
@@ -3,7 +3,15 @@ defmodule Providers.Timestamp do
   This provider implementation returns the current timestamp in UTC as an ISO8601 compliant date string.
   """
   @behaviour Providers.Provider
+
   def provide("1", _opts) do
     NaiveDateTime.utc_now() |> NaiveDateTime.to_iso8601()
   end
+
+  def provide("2", opts) do
```

Comment:
I like this and approve. However, could the same thing not be accomplished via the existing `format` field?

Reply:
That is, defer formatting until valkyrie